dev-libs/libtommath/libtommath-1.2.0.ebuild

```diff
@@ -0,0 +1,73 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools toolchain-funcs
+
+DESCRIPTION="Optimized and portable routines for integer theoretic applications"
+HOMEPAGE="http://www.libtom.net/"
+SRC_URI="https://github.com/libtom/libtommath/releases/download/v${PV}/ltm-${PV}.tar.xz"
+
+LICENSE="Unlicense"
+SLOT="1/1"
```

Comment:
I based this on [here](https://abi-laboratory.pro/index.php?view=timeline&l=libtommath), but it's worth noting that nothing actually changed wrt the soname since 1.1.0, so maybe I shouldn't change it (it was SLOT="0" there).

Reply:
This is not worth the hassle until the interface _does_ change upstream, as per devmanual.